### DISCUSSION TITLE
Error with Essentials nicknames

### DISCUSSION THREAD

**Discussion by CaptainKinchen** [EssentialsX/Essentials#4536] [Q&A] [answered]
Hello, I'm a Senior Mod on a new server and one of our players is having an issue. We have set up various ranks for the server, and for some reason they are now having a problem setting nicknames for themselves. I have brought my alternate account online to the server and set it up with identical player permissions as well as identical ranks, and the issue is not present. 

Attached is the error message the player in question receives when they try to apply any nickname. 

Note, I at one point removed their most heavily weighted role and set the same permissions from that role manually on the player specifically, and they were able to set their nickname with no issue. This first led us to assume it was an issue with the rank itself, but using the standard arrangement of ranks and permissions didn't produce the issue there, so I don't understand why I'm neither able produce the same issue on my test account nor able to resolve the issue using the working pattern on said test account.
![stigissue](https://user-images.githubusercontent.com/90926968/133830416-bf517cea-509c-4375-aa52-74d498c3ec5d.png)

**Comment by JRoy** [comment, marked as answer]
You can either remove `essentials.nick.changecolors` from your group or give `essentials.nick.changecolors.bypass` to your group.
